### ISSUE TITLE
Linter changing string content

### ISSUE THREAD

**Issue by pums974** [cphyc/fortran-syntax#8]
Hi, I'm sorry I'm faced with an other issue
the following code
```fortran
module my_module
    implicit none
contains
    subroutine my_subroutine()
        implicit none
        print*, '<test>'
        print*, "<test>"
    end subroutine my_subroutine
end module my_module
```
is transformed into
```fortran
module my_module
    implicit none
contains
    subroutine my_subroutine()
        implicit none
        print*, '<test >'
        print*, "<test >"
    end subroutine my_subroutine
end module my_module
```

I really wish that my string stayed untouched

**Comment by cphyc** [comment]
No worries, thanks for your reports!. This is a bit tricky because of the way this project has been coded (with regular expression). Essentially, we'd need a way of detecting strings and prevent them from being changed, but I don't quite know at the moment how to do this.
I think this function should be the right entry point to include a mechanism to "fence off" strings https://github.com/cphyc/fortran-syntax/blob/60080449d2d7ea7b25b1a933ddcf0fada11c2f1c/fortran_linter/main.py#L211-L232.

**Comment by cphyc** [comment]
Note that if you have some expertise in Python, pull requests are more than welcome!

**Comment by cphyc** [comment]
Just tested and your example is now properly formatted. Thank you very much for the submitted code!
This is available as of version `1.0.8` of this package.
